Penteract/Assets/Scripts/AIDuke.cpp

```diff
@@ -54,6 +54,7 @@ void AIDuke::Start() {
 }
 
 void AIDuke::Update() {
+	return;
```

Comment:
What?

Reply:
Oh, it's to play just with the bullet hell not the full bahavior, right?

Reply:
Exactly, just for development purposes, it will get reverted once bullet hell is finished.